### ISSUE TITLE
[BUG] Unable to connect to console on Steam Deck but works on MacOS

### ISSUE THREAD

**Issue by TheAntimist** [streetpea/chiaki-ng#488]
**Describe the bug**

Trying to connect to the console over the internet while I am travelling. Likely some NAT / router issues in place as Epic Games / Heroic too has issues running.

Manual IP mode works fine if I key in the specific IP, but has slightly more latency.

MacOS and PS Remote Play (iOS) works with no issues.

**Debug Log**
Attached below.

**To Reproduce**

1. Disable Manual Mode, by hitting Wifi button
2. Select PS5 with remote connection via PSN
3. See Establishing connection message.
4. Connection cancelled

**Expected behavior**

Connection should occur, as the same works on Chiaki 1.9.1 on MacOS as well.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: SteamOS
 - Version 3.6.20
 - Device: Steam Deck
 - chiaki-ng Version: 1.9.1

[chiaki_session_SteamDeck.log](https://github.com/user-attachments/files/17920668/chiaki_session_SteamDeck.log)

[Chiaki-Macos.log](https://github.com/user-attachments/files/17920644/Chiaki-Macos.log)

**Comment by streetpea** [comment]
@TheAntimist can you try this build: https://github.com/streetpea/chiaki-ng/actions/runs/12041074652/artifacts/2241899599?

To use this build you can first unzip it to your downloads folder and then make it executable with:
```
chmod +x ~/Downloads/chiaki-ng.Appimage
```
and then open it with:
```
~/Downloads/chiaki-ng.Appimage
```
Then, close it and you can copy settings from the flatpak with
```
cp ~/.var/app/io.github.streetpea.Chiaki4deck/config/Chiaki/Chiaki.conf ~/.config/Chiaki/Chiaki.conf
```
Then you can launch it again and use it.

**Comment by TheAntimist** [comment]
Thanks @streetpea ! That seems to have resolved it. How do I make it work on my regular Chiaki?

**Comment by streetpea** [comment]
@TheAntimist for some reason on your Steam deck it’s finishing the connection acceptance with the console after 2 packets. Usually it waits for three packets. Not sure why it’s doing that as the official client uses 3 packets but really only one is needed so I just changed it to that. This will be released in v1.9.2 of chiaki-ng. You could use the appimage I gave you until then. You can add it to Steam via the button at the top of the chiaki-ng menu.
